adafruit_irremote.py

```diff
@@ -309,7 +314,7 @@ def read_pulses(
         blocking=True,
         pulse_window=0.10,
         blocking_delay=0.10,
```

Comment:
All of the arguments of the read_pulses() function can be typed in the definition of that function here.